crudrest.js

```diff
@@ -18,6 +18,8 @@ var perscache = {};
 var errcb = null;
 var respcb = null; // Custom response callback
 var taidx = {}; // TODO: Move
+var Promise = require('bluebird');
```

Comment:
I don't like adding dependencies, but I know Sequelize already uses this, so it's okay (Sequelize maydepend on a different version though)